src/Resources/skeleton/authenticator/Empty.tpl.php

```diff
@@ -1,4 +1,4 @@
-<?php
+//PHP_OPEN
```

Comment:
`<?php ?>` instead?

Reply:
But then it would be evaluated by the PHP processor, and would disappear. I guess we could *always* assume we need to prepend the entire file with `<?php`. Is it worth it? They're bother weird honestly...

Reply:
> we could always assume we need to prepend the entire file

looks like a good idea to me

Reply:
always prepending is bad IMO: a generator may have a case for generating another type of file.

another option might be this (if it works):

```php
<?= '<?php' ?>
```

Reply:
What about `//<?php`?